rj_gameplay/stp/skill/ball_carrier.py

```diff
@@ -0,0 +1,37 @@
+from abc import ABC, abstractmethod
+from typing import Optional, Type
+
+import numpy as np
+
+import stp.skill as skill
+from stp import role as role
+
+
+class Pass:
+    __slots__ = ["passer", "receiver", "pt"]
+
+    passer: int
+    receiver: int
+    pt: np.ndarray
+
+    def __init__(self, passer: int, receiver: int, pt: np.ndarray):
```

Comment:
Can we make a `RobotId` alias for `int`? Been meaning to do that C++-side.

Reply:
Done